spec/classes/snmp_client_spec.rb

```diff
@@ -36,17 +36,15 @@
           :operatingsystemrelease => '6.4'
         }
         end
-        it { should contain_package('snmp-client').with(
-          :ensure => 'present',
-          :name   => 'net-snmp-utils'
-        )}
+
+        it { should contain_package('snmp-client').that_comes_before('File[snmp.conf]') }
+
```

Comment:
This removes some testing functionality.  It may be better to just add to the `:require` below.